### PR TITLE
Add docs on how to change the configuration

### PR DESCRIPTION
**Pull request by ahg-g**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Documents how to change the default configuration of a released version

#### Which issue(s) this PR fixes:
Part of https://github.com/kubernetes-sigs/kueue/issues/169

### CODE REVIEW

**Comment by ahg-g** [comment]
cc @cortespao

File: docs/setup/install.md [hunk 1]
```diff
@@ -17,6 +17,27 @@ VERSION=v0.1.0
 kubectl apply -f https://github.com/kubernetes-sigs/kueue/releases/download/$VERSION/manifests.yaml
 ```
 
+## Install a custom-configured released version
+
+To install a custom-configured released version of Kueue in your cluster, execute the following steps: 
+
+First, download the release's `manifests.yaml` file:
```
Comment (alculquicondor) [inline docs/setup/install.md:24]:
```suggestion
1. Download the release's `manifests.yaml` file using the following command:
```

File: docs/setup/install.md [hunk 2]
```diff
@@ -17,6 +17,27 @@ VERSION=v0.1.0
...
+
+```shell
+VERSION=v0.1.0
+wget https://github.com/kubernetes-sigs/kueue/releases/download/$VERSION/manifests.yaml
+```
+
+Second, with your favourite editor open `manifests.yaml` and edit the `kueue-manager-config` 
+ConfigMap manifest, specifically edit the `controller_manager_config.yaml` data entry which represents
```
Comment (alculquicondor) [inline docs/setup/install.md:32]: Probably worth an example?

Comment (ahg-g) [reply docs/setup/install.md:32]: should we add that in a separate config.md file?

Comment (ahg-g) [reply docs/setup/install.md:32]: similar to https://kubernetes.io/docs/reference/scheduling/config/

Comment (alculquicondor) [reply docs/setup/install.md:31]: That would make sense as we add more configuration complexity in the future. But for now, I'm just not sure the way you are referring to the data entry is clear enough. So an example would clarify what users should look for.

Maybe there could be a phrase such as:

```
The contents of the ConfigMap are similar to the following:
```

File: docs/setup/install.md [hunk 3]
```diff
@@ -17,6 +17,26 @@ VERSION=v0.1.0
...
+
+```shell
+VERSION=v0.1.0
+wget https://github.com/kubernetes-sigs/kueue/releases/download/$VERSION/manifests.yaml
+```
+2. Using your favourite editor, open `manifests.yaml` and edit the `kueue-manager-config` 
+ConfigMap manifest, specifically edit the `controller_manager_config.yaml` data entry which represents
+the default Kueue Configuration struct ([v1alpha1@v0.1.0](https://pkg.go.dev/sigs.k8s.io/kueue@v0.1.0/apis/config/v1alpha1#Configuration)).
+
+3. Apply the customized manifests to the cluster:
```
Comment (alculquicondor) [inline docs/setup/install.md:34]:
```suggestion
3. To apply the customized manifests to the cluster, run the following command:
```

File: docs/setup/install.md (hunk 2 above)
Comment (ahg-g) [reply docs/setup/install.md:31]: > That would make sense as we add more configuration complexity in the future

My thinking is that we need to emphasize the importance of the `manageJobsWithoutQueueName` flag. May be we should just leave it to the API docs?

File: docs/setup/install.md [hunk 4]
```diff
@@ -18,6 +18,38 @@ VERSION=v0.1.0
...
+
+```shell
+VERSION=v0.1.0
+wget https://github.com/kubernetes-sigs/kueue/releases/download/$VERSION/manifests.yaml
+```
+2. Using your favourite editor, open `manifests.yaml` and edit the `kueue-manager-config` 
+ConfigMap manifest, specifically edit the `controller_manager_config.yaml` data entry which represents
+the default Kueue Configuration 
+struct ([v1alpha1@v0.1.0](https://pkg.go.dev/sigs.k8s.io/kueue@v0.1.0/apis/config/v1alpha1#Configuration)).
+The contents of the ConfigMap are similar to the following:
```
Comment (alculquicondor) [inline docs/setup/install.md:35]: These are the contents of a field of the ConfigMap, not the ConfigMap :)

I think we can just document the ConfigMap here, and leave the contents of the Configuration to a separate config doc.

Comment (ahg-g) [reply docs/setup/install.md:35]: ok, but we still need to put a sample configuration, see how it looks now.

**Review by alculquicondor — COMMENTED** [review]
Looks good. I'll leave the rest to @cortespao

File: docs/setup/install.md [hunk 5]
```diff
@@ -18,6 +18,47 @@ VERSION=v0.1.0
...
+
+To install a custom-configured released version of Kueue in your cluster, execute the following steps: 
+
+1. Download the release's `manifests.yaml` file using the following command:
+
+```shell
+VERSION=v0.1.0
+wget https://github.com/kubernetes-sigs/kueue/releases/download/$VERSION/manifests.yaml
+```
+2. Using your favourite editor, open `manifests.yaml` and edit the `kueue-manager-config`
```
Comment (cortespao) [inline docs/setup/install.md:31]:
```suggestion
2. With an editor of your preference, open `manifests.yaml`.
3. In the `kueue-manager-config` 
```

File: docs/setup/install.md [hunk 6]
```diff
@@ -18,6 +18,47 @@ VERSION=v0.1.0
...
+    health:
+      healthProbeBindAddress: :8081
+    metrics:
+      bindAddress: :8080
+    webhook:
+      port: 9443
+    manageJobsWithoutQueueName: true
+```
+
+3. To apply the customized manifests to the cluster, run the following command:
```
Comment (cortespao) [inline docs/setup/install.md:56]:
```suggestion
3. Apply the customized manifests to the cluster:
```

File: docs/setup/install.md [hunk 7]
```diff
@@ -18,6 +18,47 @@ VERSION=v0.1.0
...
+To install a custom-configured released version of Kueue in your cluster, execute the following steps: 
+
+1. Download the release's `manifests.yaml` file using the following command:
+
+```shell
+VERSION=v0.1.0
+wget https://github.com/kubernetes-sigs/kueue/releases/download/$VERSION/manifests.yaml
+```
+2. Using your favourite editor, open `manifests.yaml` and edit the `kueue-manager-config` 
+ConfigMap manifest, specifically edit the `controller_manager_config.yaml` data entry which represents
```
Comment (cortespao) [inline docs/setup/install.md:32]:
```suggestion
ConfigMap manifest, edit the `controller_manager_config.yaml` data entry. This entry represents
```

**Review by cortespao — COMMENTED** [review]
Small suggestions

File: docs/setup/install.md [hunk 8]
```diff
@@ -18,6 +18,47 @@ VERSION=v0.1.0
 kubectl apply -f https://github.com/kubernetes-sigs/kueue/releases/download/$VERSION/manifests.yaml
 ```
 
+## Install a custom-configured released version
```
[the remark this answers is not in the document]
Comment (ahg-g) [reply docs/setup/install.md:21]: No really, it is custom "configured", the configuration is the custom, not the release itself.

File: docs/setup/install.md [hunk 9]
```diff
@@ -18,6 +18,47 @@ VERSION=v0.1.0
...
+
+To install a custom-configured released version of Kueue in your cluster, execute the following steps: 
+
+1. Download the release's `manifests.yaml` file using the following command:
```
[the remark this answers is not in the document]
Comment (ahg-g) [reply docs/setup/install.md]: done.

File: docs/setup/install.md (hunk 7 above)
Comment (ahg-g) [reply docs/setup/install.md]: done.

File: docs/setup/install.md (hunk 6 above)
Comment (ahg-g) [reply docs/setup/install.md]: done.

**Review by alculquicondor — COMMENTED** [review]
/label merge-method/squash

**Pull request merged**